docs/dev/table/functions.md

```diff
@@ -1230,6 +1230,17 @@ COT(numeric)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+COTH(numeric)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the hyperbolic cotangent of a <i>numeric</i>.</p>
```

Comment:
please add some information about returned types, refer to @pnowojski 's review suggestion in #6700